index.html

```diff
@@ -54,8 +57,189 @@ <h3>I'm a Software Developer</h3>
         </ul>
       </div>
     </section>
-  </div>
+    <section class="recent_works padding_lr">
+      <div class="title">
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="place_holder">
+        <img src="/static/images/ph1.png" alt="place_holder">
+      </div>
+      <div class="mp_stories">
+        <h2>Multi-Post Stories</h2>
+        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required: has been
+          the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div class="links">
+        <ul>
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons btn_width_wrap">
+        <a href="#">See project</a>
+        .
+      </div>
+    </section>
```

Comment:
- [OPTIONAL] Instead of using the `section` tag, for the individual projects, I would advise you to use the `article` tag. The `section` tag defines sections in a document, such as chapters, headers, footers, or any other sections of the document while the `article` tag specifies independent, self-contained content. I would advise you to use this HTML structure for your portfolio:

```HTML
<header>
  <a href="index.html">logo</a>
  <nav>
    hamburger menu
  </nav>
</header>
<section class="headline">
  <h1>Name of Developer</h1>
  <h2>I'm a developer</h2>
  <p>description</p>
  ...
</section>
<section class="projects">
  <article class="project_1">
    ...
  </article>
  <article class="project_2">
    ...
  </article>
  <article class="project_3">
    ...
  </article>
</section>
<setion class="about_me">
  <h2>About me</h2>
  <p>description</p>
  <a>Get My Resume</a>
  <ul class="languages">
    <h3>Languages</h3>
    <ul>
      <li>language_1</li>
      <li>language_2</li>
      <li>language_3</li>
    </ul>
  </ul>
  <ul class="skills">
    <h3>Skills</h3>
    <ul>
      <li>skill_1</li>
      <li>skill_2</li>
      <li>skill_3</li>
    </ul>
  </ul>
  ...
</setion>
```

Reply:
Hi @wacoo, Although this is optional, I am advising you to adopt the structure that I gave you. 👍

Reply:
I won't change it for now. It messes with the layout. I will use it in the future thought. Thanks for the help.